### PR TITLE
'classes'

### PR DESCRIPTION
**Pull request by dbulka**

### CODE REVIEW

File: controller/order.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Order:
+
+    def __init__(self, id_order):
+        self.id_order = id_order
+
+    def calc_ticket(self, *args,  **kwargs)
+
+    def calc_flight_settings(self, *args,  **kwargs)
```
Comment (shukal94) [inline controller/order.py]: blank line at EOF please

File: controller/order.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Order:
+
+    def __init__(self, id_order):
+        self.id_order = id_order
+
+    def calc_ticket(self, *args,  **kwargs)
```
Comment (shukal94) [inline controller/order.py]: add : and description
def do(*args, **kwargs):
   """ lllalala """

File: flight_settings/business.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from flight_settings import Flight
+
+class BusinessFlight(Flight)
```
Comment (shukal94) [inline flight_settings/business.py]: it's uniterpretable code, fix it, and dont forget about line at EOF

File: flight_settings/common.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from flight_settings import Flight
+
+class CommonFlight(Flight)
```
Comment (shukal94) [inline flight_settings/common.py]: same

File: flight_settings/flight.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class Flight:
+
+    def __init__(self, *args, **kwargs)
+
+    def choose_class(self, *args, **kwargs)
+
+    def choose_luggage(self, *args, **kwargs)
```
Comment (shukal94) [inline flight_settings/flight.py]: the same as previvous

File: model/admin.py [hunk 6]
```diff
@@ -0,0 +1,6 @@
+from user import User
+
+
+class Admin(User):
+
+    def make_changes(self, *args, **kwargs)
```
Comment (shukal94) [inline model/admin.py]: see above

File: model/customer.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class Guest(User):
+
+    def __init__(self, id):
+        self.id = id
+
+    def book_ticket(self):
+
+    def buy_ticket(self):
```
Comment (shukal94) [inline model/customer.py]: add `pass` or docstring into methods body

File: model/guest.py [hunk 8]
```diff
@@ -0,0 +1,6 @@
+from user import User
+
+class Guest(User):
+
+    def __init__(self, id):
+        self.id = id
```
Comment (shukal94) [inline model/guest.py]: blank line at EOF

File: model/user.py [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+    def check_person_info(self, *args, **kwargs)
+
+    def check_in(self, *args, **kwargs)
+
+    def register(self, *args, **kwargs)
+
+    def search_flight(self, *args, **kwargs)
+
+    def order_ticket(self, *args, **kwargs)
```
Comment (shukal94) [inline model/user.py]: see above

File: ticket/child_ticket.py [hunk 10]
```diff
@@ -0,0 +1,3 @@
+from ticket import Ticket
+
+class ChildTicket(Ticket)
```
Comment (shukal94) [inline ticket/child_ticket.py]: see above

File: ticket/full_ticket.py [hunk 11]
```diff
@@ -0,0 +1,4 @@
+from ticket import Ticket
+
+class FullTicket(Ticket)
```
Comment (shukal94) [inline ticket/full_ticket.py]: see above

File: ticket/ticket.py [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+
+    def apply_direction(self, *args, **kwargs)
+
+    def choose_class(self, *args, **kwargs)
+
+    def change_luggage(self, *args, **kwargs)
+
+    def change_place(self, *args, **kwargs)
+
+    def change_menu(self, *args, **kwargs)
```
Comment (shukal94) [inline ticket/ticket.py]: see above

**Review by shukal94 — CHANGES_REQUESTED** [review]
please, read pep-8 and basic methods definition

**Review by shukal94 — CHANGES_REQUESTED** [review]
good, but double check about blank lines at end of file

**Comment by shukal94** [comment]
and place these files accordingly to init struct

**Review by shukal94 — APPROVED** [review]
Good job

**Review by zipziprar — APPROVED** [review]

**Pull request merged**